src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeMultiFetch.scala

```diff
@@ -71,6 +73,21 @@ object TezosNodeMultiFetch {
     def decodeJson = multiFetch.decodeJson.product(additionalDecode)
   }
 
+  /** Combines two multifetch acting on the same inputs to get a pair of the outputs,
+   * considering that the intermediate json should not be the same, as required for `addDecoding`
+   * TODO consider composing with a different function that (,) as to combine more than one with different out shapes
+   */
+  def tupleResults[F[_]: Monad, T[_] : Traverse: FunctorFilter, I, O1, O2, J1, J2](mf1: Aux[F, T, I, O1, J1], mf2: Aux[F, T, I, O2, J2]): Kleisli[F, T[I], T[(I, (O1, O2))]] =
```

Comment:
What use case will we need an input type to be mapped to two output types for a single instance of Json?

Reply:
I've added documentation with examples that should clarify why I needed these combinators